tests/unit/models/test_base_model.py

```diff
@@ -82,15 +83,44 @@ def test_is_dict_of_model_when_not_true(self):
             ),
         )
 
+    def test_bad_type(self):
+        transaction_dict = {
+            "account": 1,
+            "amount": 10,
+            "destination": 1,
+        }
+        with self.assertRaises(XRPLModelException):
+            Payment(**transaction_dict)
+
+    def test_bad_type_flags(self):
+        transaction_dict = {
+            "account": 1,
+            "amount": 10,
+            "destination": 1,
+            "flags": "1234",
+        }
+        with self.assertRaises(XRPLModelException):
+            Payment(**transaction_dict)
```

Comment:
I feel it's better to isolate the bad type to a single attribute, instead of coalescing with the previous test case 
```suggestion
            Payment(account="source_acct", destination="dest_acct", flags="1234", amount='200')
```
```suggestion
            Payment(**transaction_dict)
```